LongWarOfTheChosen/Localization/LW_Overhaul.ita

```diff
@@ -4,195 +4,195 @@ GrazeBandWidthMod="Zona graffio"
 GrazeBandWidthModTooltip="Determina quanto deve essere vicino un colpo al centro esatto per contare come graffio"
 GrazeBandValue="Valore: <XGParam:IntValue0>"
 PauseOnRecruitMod="Pausa al reclutamento"
-PauseOnRecruitModTooltip="Pausa sul planisfero quando un rifugio recluta un nuovo soldato o un ribelle"
+PauseOnRecruitModTooltip="Pausa sul planisfero quando un Rifugio recluta un nuovo soldato o un ribelle"
 
 [UIScreenListener_ShellDifficulty]
-strIncompatibleModsTitle="MOD DUPLICATI PRESENTI"
-strIncompatibleModsText="Rilevati mod duplicati:\n%modnames%\nQuesti mod sono stati integrati in questo mod e dovrebbero essere disabilitati dal launcher. Caricandolo potrebbero verificarsi effetti negativi, come scarse prestazioni o instabilità. Procedere a proprio rischio."
+strIncompatibleModsTitle="MOD DUPLICATE PRESENTI"
+strIncompatibleModsText="Rilevate mod duplicate:\n%modnames%\nQueste mod sono già state integrate in questa mod e dovrebbero essere disabilitate dal launcher. Caricandole potrebbero verificarsi effetti negativi, come scarse prestazioni o instabilità. Procedere a proprio rischio."
 
 [UIMission_LWCustomMission]
 m_strMissionDifficulty_start="ATTIVITÀ NEMICA BASE"
 
 [ProtectRegionEarly X2LWAlienActivityTemplate]
 ActivityName="ProtectRegion1Activity"
-ActivityObjectives[0]="Trova un indizio"
+ActivityObjectives[0]="Inizia a liberare la regione"
```

Comment:
Please wrap this line with the following block, as described in the wiki:
```
; LWOTC Translated
...
; End Translated
```
It's a clunky way of tracking what has already been translated, but it should work better than just trying to see whether the translation matches the current text of the English-language version.